cmd/bad-key-revoker/main.go

```diff
@@ -357,10 +362,16 @@ func main() {
 			// keyHashToSerial table at once
 			FindCertificatesBatchSize int
 
-			// Interval specifies how long bad-key-revoker should sleep between attempting to find
-			// blockedKeys rows to process when there is no work to do
+			// Interval specifies the minimum duration bad-key-revoker
+			// should sleep between attempting to find blockedKeys rows to
+			// process when there is no work to do.
 			Interval cmd.ConfigDuration
```

Comment:
I am in favor of removing this completely and setting a default of 1 second. I don't think it is particularly useful as a configurable option when using backoff.